tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- set_fact:
+    ssh_dir_prefix: "{{ '/home' if ssh_dir_owner != 'root' }}"
+
+- set_fact:
+    ssh_dir_path: "{{ ssh_dir_prefix }}/{{ ssh_dir_owner }}/.ssh"
+
+- set_fact:
+    ssh_key_path: "{{ ssh_dir_path }}/{{ ssh_key_name }}"
+
+- include_tasks: create.yml
+  when: ssh_key_install
+
+- include_tasks: remove.yml
+  when: not ssh_key_install
```

Comment:
I think this could lead to confusion because I wouldn't associate the lack of installation with removal. How about we rename `ssh_key_install` -> `ssh_key_action` with a default of `install` but with an alternative value of `remove`? We can add an assertion that's it's one or the other prior to execution